.circleci/config.yml

```diff
@@ -27,6 +27,9 @@ jobs:
     steps:
       - restore_cache:
           key: protocol-{{ .Environment.CIRCLE_SHA1 }}
+      - run:
+          name: Install Prereqs
+          command: sudo apt-get install -y rsync
```

Comment:
What was this for?

Reply:
We need to rsync the typechain types files on top of the types generated by tsc. See the build process in core/package.json.

Reply:
I only see that you added the `build` script for the `common` package, I don't see `rsync`?

Reply:
Which means we need to install the rsync package.